### PR TITLE
Updated database and server-api documents again.

### PR DESCRIPTION
**Pull request by gwitmond**

### CODE REVIEW

File: notes/database.md [hunk 1]
```diff
@@ -41,6 +39,12 @@ char-object (inside page):
  - `y2` (int, right) in pixels
  - `c` (string, character)
 
+images
```
Comment (GeertJohan) [inline notes/database.md:42]: I believe this collection to be obsolete, as the GridFS filename is predictable: `images/documentId-pageNr.png`

**Comment by gwitmond** [comment]
You are correct in that.

However, from a data modeling perspective I like to have all data available in the model. And not hidden behind an assumption only found in the code.

**Comment by GeertJohan** [comment]
There's a clear difference between assumptions and "should be like".
I opt for a section defining how files in GridFS should be named, thereby removing the 'assumption' and making it a requirement.

**Comment by gwitmond** [comment]
Top. Write that section and my concerns have been addressed.

**Pull request merged**